### PR TITLE
Updating dependencies

### PR DESCRIPTION
**Pull request by zendranm**
This PR updates the following dependencies:
- typescript to v4.8.4
- eslint-config-vazco to 7.0.0

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "@types/sinon": "10.0.11",
     "@types/sinon-chai": "3.2.8",
     "@types/strip-json-comments": "3.0.0",
-    "eslint-config-vazco": "6.2.0",
-    "typescript": "4.6.2"
+    "eslint-config-vazco": "^7.0.0",
+    "typescript": "^4.8.4"
```
Comment (radekmie) [inline package.json:21]: Let's use pinned versions.

**Review by radekmie — APPROVED** [review]

**Review by radekmie — APPROVED** [review]

File: tsconfig.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "include": ["source/common.ts"],
+  "include": ["source/**.*"],
```
Comment (radekmie) [inline tsconfig.json:2]: I think it's enough to say `"source"`.

File: package.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
 {
   "private": true,
   "scripts": {
-    "lint:js": "eslint .",
-    "lint:ts": "tsc",
+    "lint:js": "eslint source/",
+    "lint:ts": "tsc --noEmit source/common.ts",
```
Comment (radekmie) [inline package.json:5]: 1. Why only `source` get's linted?
2. Why only `source/common.ts` get's `tsc`ed?

**Review by radekmie — COMMENTED** [review]
Both `lint:js` and `lint:ts` now work.

File: source/server.ts [hunk 4]
```diff
@@ -1,12 +1,12 @@
 import type { NextHandleFunction } from 'connect';
-import Fibers from 'fibers';
-import YAML from 'js-yaml';
+import { current } from 'fibers';
```
Comment (radekmie) [inline source/server.ts:2]: This is not correct, as `current` get's overridden in the runtime, therefore we have to access it every time.

**Review by radekmie — CHANGES_REQUESTED** [review]

**Review by piotrpospiech — APPROVED** [review]

**Pull request merged**